tools/appsre-build-worker-packer.sh

```diff
@@ -2,38 +2,35 @@
 # AppSRE runs this script to build an ami and share it with an account
 set -exv
 
+if [ -z "$COMMIT_SHA" ]; then
+    COMMIT_SHA=$(git rev-parse HEAD)
+fi
 
-COMMIT_SHA=$(git rev-parse HEAD)
-COMMIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-ON_JENKINS=true
-SKIP_CREATE_AMI=false
-BUILD_RPMS=false
+if [ -z "$COMMIT_BRANCH" ]; then
+    COMMIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+fi
 
-# Use gitlab CI variables if available
-if [ -n "$CI_COMMIT_SHA" ]; then
-    ON_JENKINS=false
-    COMMIT_SHA="$CI_COMMIT_SHA"
+if [ -z "$SKIP_CREATE_AMI" ]; then
+    SKIP_CREATE_AMI=false
 fi
-if [ -n "$CI_COMMIT_BRANCH" ]; then
-    COMMIT_BRANCH="$CI_COMMIT_BRANCH"
-elif [ -n "$GIT_BRANCH" ]; then
-    # Use jenkins CI variables if available
-    COMMIT_BRANCH="${GIT_BRANCH#*/}"
+
+if [ -z "$BUILD_RPMS" ]; then
+    # Build RPMs when building RHEL on AppSRE's infra
+    BUILD_RPMS=true
 fi
 
-if [ "$ON_JENKINS" = false ]; then
-    # work around not working podman from 9.1 on a 9.0 image
-    # see https://bugzilla.redhat.com/show_bug.cgi?id=2143282
-    # TODO: Remove me when the bug is fixed or we switch to 9.1
-    sudo dnf remove -y python-unversioned-command
-    sudo dnf upgrade -y
+if [ -z "$SKIP_TAGS" ]; then
+    SKIP_TAGS="rpmrepo"
+fi
 
-    sudo dnf install -y podman jq
+if [ -z "$PACKER_ONLY_EXCEPT" ]; then
+    # Build rhel only
+    PACKER_ONLY_EXCEPT=--only=amazon-ebs.rhel-9-x86_64,amazon-ebs.rhel-9-aarch64
 fi
```

Comment:
nitpick: you could have used just the Bash default value assignment instead of `if` statements.

So e.g.
```bash
COMMIT_SHA=${COMMIT_SHA:-$(git rev-parse HEAD)}
```
and so on. It would be a bit shorter...